shoop/core/pricing/_discounts.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+import abc
+
+import six
+
+from shoop.apps.provides import load_module_instances
+
+
+def get_discount_modules():
+    """
+    Get a list of configured discount module instances.
+
+    :rtype: list[DiscountModule]
+    """
+    return load_module_instances("SHOOP_DISCOUNT_MODULES", "discount_module")
+
+
+class DiscountModule(six.with_metaclass(abc.ABCMeta)):
```

Comment:
I don't think there's any sense (aside from a purely logical viewpoint) in the term "negative discount".